app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/ProfileViewModel.kt

```diff
@@ -22,7 +23,7 @@ data class SongList(val name: String, val tracks: List<Track> = mutableListOf())
 class ProfileViewModel
 @Inject
 constructor(
-    private val profileRepository: ProfileRepository,
+    private val repository: ProfileRepository, // TODO revoir
```

Comment:
Not sure what you wanted to double check here but I'll leave this comment as a reminder just in case